error: variable 'toggle_len' set but not used [-Werror,-Wunused-but-set-variable]

```
/usr/bin/c++  -I/wrkdirs/usr/ports/audio/ncpamixer/work/.build -I/wrkdirs/usr/ports/audio/ncpamixer/work/ncpamixer-1.3.3.1/src -I/wrkdirs/usr/ports/audio/ncpamixer/work/ncpamixer-1.3.3.1/src/src -I/usr/local/include -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -Werror -Wall -Wpedantic -std=gnu++14 -MD -MT CMakeFiles/ncpamixer.dir/ui/tab.cpp.o -MF CMakeFiles/ncpamixer.dir/ui/tab.cpp.o.d -o CMakeFiles/ncpamixer.dir/ui/tab.cpp.o -c /wrkdirs/usr/ports/audio/ncpamixer/work/ncpamixer-1.3.3.1/src/ui/tab.cpp
/wrkdirs/usr/ports/audio/ncpamixer/work/ncpamixer-1.3.3.1/src/ui/tab.cpp:422:13: error: variable 'toggle_len' set but not used [-Werror,-Wunused-but-set-variable]
        int toggle_len = 0;
            ^
1 error generated.
```

Version: 1.3.3.1
OS: FreeBSD 14

--- a/src/ui/tab.cpp
+++ b/src/ui/tab.cpp
@@ -419,8 +419,6 @@ void Tab::handleMouse(int x, int y, int button)
             }
         }
 
-        int toggle_len = 0;
-        int item_len = 0;
         unsigned int len = 0;
         unsigned int sink_pos = 0;
         PaObject* toggle_item = nullptr;
@@ -434,7 +432,6 @@ void Tab::handleMouse(int x, int y, int button)
                 if (name != nullptr) {
                     len = strlen(name);
                     sink_pos = ui.width - 1 - len;
-                    item_len = strlen(name);
                     toggle_item = rel->second;
                 }
             }
@@ -442,7 +439,6 @@ void Tab::handleMouse(int x, int y, int button)
             if (i.second->active_attribute != nullptr && has_volume) {
                 len = strlen(i.second->active_attribute->description);
                 sink_pos = ui.width - 1 - len;
-                item_len = strlen(i.second->active_attribute->description);
                 toggle_item = i.second;
             }
         }
@@ -462,8 +458,6 @@ void Tab::handleMouse(int x, int y, int button)
             if (done) {
                 return;
             }
-
-            toggle_len += item_len;
         }
 
         baseY += BLOCK_SIZE;

